docs/source/getting-started/content/assembly_troubleshooting.adoc

```diff
@@ -1,4 +1,11 @@
 [id="troubleshooting-codeready-containers_{context}"]
 = Troubleshooting {rh-prod}
 
+[NOTE]
+====
+{rh-prod}'s goal is to deliver an OpenShift environment for development and testing purposes.
```

Comment:
Apologies, I missed this before. We should try to avoid CodeReady Containers showing possession of anything, so I'd reword this slightly:

```adoc
The goal of {rh-prod} is to....
```

Beyond that, this PR looks good to me. :+1: